gitoid/src/gitoid.rs

```diff
@@ -193,11 +221,12 @@ fn bytes_from_buffer<R>(
     mut digester: Box<dyn DynDigest>,
     mut reader: BufReader<R>,
     expected_length: usize,
+    object_type: ObjectType,
 ) -> Result<(usize, [u8; NUM_HASH_BYTES])>
 where
     BufReader<R>: Read,
 {
-    let prefix = format!("blob {}\0", expected_length);
+    let prefix = format!("{} {}\0", object_type, expected_length);
```

Comment:
Fixed a bug where the prior code didn't actually generate any non-blob gitoids correctly.